src/olympia/reviewers/views.py

```diff
@@ -975,7 +998,7 @@ def reviewlog(request):
     return render(request, 'reviewers/reviewlog.html', data)
 
 
-@addons_reviewer_required
+@any_reviewer_required
```

Comment:
Not sure about this one, but it feels like any reviewer is a bit broad.

Reply:
The link to it is present on the review page, so it might be useful to content reviewers, post reviewers and legacy reviewers... so, pretty much any reviewer really...

Reply:
persona/theme reviewers :)
But I don't care.